access_card/src/main/res/layout/layout_member_access_card.xml

```diff
@@ -34,14 +35,28 @@
     <TextView
         android:id="@+id/expiration"
         style="@style/BodySansSerifBlackLeft"
-        android:layout_width="wrap_content"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginStart="@dimen/marginDouble"
-        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_marginEnd="@dimen/marginDouble"
+        android:lines="1"
+        app:layout_constraintEnd_toStartOf="@id/reciprocalMember"
         app:layout_constraintHorizontal_bias="0"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/membershipType"
-        tools:text="Member Plus" />
+        tools:text="Member Plus Member Plus Member Plus Member Plus" />
+
+    <ImageView
+        android:id="@+id/reciprocalMember"
```

Comment:
Shouldn't the default visibility be `GONE`? How are we hiding it if the membership is not reciprocal?

Reply:
Yes, it should be `GONE`. Updated.